core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -107,6 +107,9 @@ var ExplorationEditorPage = function() {
   var saveDiscardToggleButton = element(
     by.css('.protractor-test-save-discard-toggle'));
   var saveDraftButton = element(by.css('.protractor-test-save-draft-button'));
```

Comment:
commitChangesButton (so that it's not so close to "save draft" or "publish changes")

Reply:
Done